### ISSUE TITLE
Linked List [C++]

### ISSUE THREAD

**Issue by S2606** [iiitv/algos#377]
#### This issue is about [Linked List](https://en.wikipedia.org/wiki/Linked_list).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by S2606** [comment]
@aviaryan @singhpratyush Can you please assign this to me ?

**Comment by devshiva619** [comment]
@S2606 what do you want to be done in this issue?

**Comment by S2606** [comment]
@devshiva619 If you wanna do this, you can go ahead.. 👍

**Comment by devshiva619** [comment]
@S2606 I would surely work but I am not able to understand what work is to be done? Tell me the work to be done and I will start with this issue.

**Comment by devshiva619** [comment]
@S2606 can you please tell what's need to be done so that I can proceed with the work

**Comment by Monal5031** [comment]
@devshiva619 You have to implement Linked List in `cpp`.

**Comment by devshiva619** [comment]
@Monal5031 thanks for help :)

**Comment by devshiva619** [comment]
@Monal5031 so do I need to have same function names as other programs?

**Comment by Monal5031** [comment]
@devshiva619 Yes its mentioned in contributing guidelines that function name in which the core algorithm is present is to be named as same that of file name.

**Comment by himanshusanecha** [comment]
if possible i want to work on this issue @iiitv/project-maintainers

**Comment by yashshah2820** [comment]
@himanshusanecha GFI 👍
